cfg/validate.go

```diff
@@ -177,6 +177,9 @@ func isValidMetricsConfig(m *MetricsConfig) error {
 	if m.StackdriverExportInterval != 0 && m.CloudMetricsExportIntervalSecs != 0 {
 		return fmt.Errorf("exactly one of stackdriver-export-interval and cloud-metrics-export-interval-secs must be specified")
 	}
+	if m.PrometheusPort > 65535 {
```

Comment:
Shouldn't we declare a constant for 65535? Also, we should print the prometheus-port received from user in the error.

Reply:
Yes. Thanks